lib/shoryuken/polling.rb

```diff
@@ -0,0 +1,88 @@
+module Shoryuken
+  module Polling
+    QueueConfiguration = Struct.new(:name, :options) do
+      def hash
+        name.hash
+      end
+
+      def ==(other)
```

Comment:
I think we should go without it. It's weird. `==` is very tricky, tricky == are even more trickier :).